src/Orleans.Connections.Security/Security/TlsClientConnectionMiddleware.cs

```diff
@@ -65,15 +66,15 @@ private async Task InnerOnConnectionAsync(ConnectionContext context)
                 leaveOpen: true
             );
 
-            SslDuplexPipe sslDuplexPipe = null;
+            TlsDuplexPipe sslDuplexPipe = null;
```

Comment:
minor but it should be renamed to `tlsDuplexPipe` now

Reply:
Thanks, fixed